server/src/main/java/org/opensearch/cluster/node/DiscoveryNodeRole.java

```diff
@@ -193,15 +193,40 @@ public final String toString() {
 
     /**
      * Represents the role for a master-eligible node.
+     * @deprecated As of 2.0, because promoting inclusive language, replaced by {@link #CLUSTER_MANAGER_ROLE}
      */
+    @Deprecated
     public static final DiscoveryNodeRole MASTER_ROLE = new DiscoveryNodeRole("master", "m") {
 
+        @Override
+        public Setting<Boolean> legacySetting() {
+            // copy the setting here so we can mark it private in org.opensearch.node.Node
+            // As of 2.0, set the default value to 'false', so that MASTER_ROLE isn't added as a default value of NODE_ROLES_SETTING
+            return Setting.boolSetting("node.master", false, Property.Deprecated, Property.NodeScope);
+        }
+
+    };
+
+    /**
+     * Represents the role for a cluster-manager-eligible node.
+     */
+    public static final DiscoveryNodeRole CLUSTER_MANAGER_ROLE = new DiscoveryNodeRole("cluster_manager", "m") {
+
         @Override
         public Setting<Boolean> legacySetting() {
             // copy the setting here so we can mark it private in org.opensearch.node.Node
             return Setting.boolSetting("node.master", true, Property.Deprecated, Property.NodeScope);
         }
 
+        @Override
+        public DiscoveryNodeRole getCompatibilityRole(Version nodeVersion) {
+            if (nodeVersion.onOrAfter(Version.CURRENT)) {
```

Comment:
Shouldn't it be .Version.2_0_0? Current seems to be a moving target ...

Reply:
This is a good catch. 😄 Thank you. I thought about using `Version.2_0_0`, but then got confused. I will change the Version here.

Reply:
Corrected the version in the commit https://github.com/opensearch-project/OpenSearch/pull/2424/commits/83a3c733170c32b1f939083f14f26c41993e4f16

Reply:
:+1: